### DISCUSSION TITLE
Disabling VST with AG connected to Kontakt Instrument looses settings (in Nuendo)

### DISCUSSION THREAD

**Discussion by ankarfeldt** [apohl79/audiogridder#492] [General]
Really cool to see this project moving forward. 

I realize AG was probably first designed with Audio Effect plugins in mind, but the idea of loading VST instruments are so appealing to me. Then you could have an entire (orchestra) template with AG plugins, maybe with the tracks disabled by default, and it would actually make the template completely flexible within each project. That would really be something and I would gladly replace my VEP setup for the ease of routing and access to each sound. The way I would mix the template would also change. 

So I've already done some tests and been impressed with how well AG can handle a lot of streams on the server side (almost seemingly better performance then VEP), had 30 instances of AG running Vienna Instrument Pro blasting more then 1000 "voices" at the same time. Still a long way to go to see how AG reacts on 200 instruments on a server and the DAW playing a piece normally. 
I did experienced some instability already, especially if there were any changes to the latency in the DAW or in the AG plugin. Will have to report back on that later when I've done more testing. 

One issue though that I currently found was loading a Kontakt plugin and adding a NKI patch (kontakt instrument), afterwards if I have disable that AG instance and then enable it again, the Kontakt plugin has lost its settings including the NKI patch. This happens both with AU, VST and Kontakt 5 & 6. 
This is done in Nuendo 11. I also tried saving and loading a Track Preset which also lost the settings. 
In Logic Pro I tried also saving and loading a track preset, or freezing and unfreezing the track. All of which also lost the settings. 
This doesn't happen with Vienna Instrument Pro or Massive, which are the only two other VST instruments I've tested. But it seems clear that it has something to do with Kontakt :) 

I'll start donating as I'd love for this project to grow. Best

**Comment by apohl79** [comment]
Great stuff! :-)

Regarding loosing settings: Can you try activating "always" syncing? (In the plugin, click the settings button -> "Remote Sync Frequency" -> "Always (every 10s)" - and see if that helps?
